homeassistant/components/hue_ble/__init__.py

```diff
@@ -0,0 +1,52 @@
+"""Hue BLE integration."""
+
+import logging
+
+from HueBLE import HueBleLight
+
+from homeassistant.components.bluetooth import (
+    async_ble_device_from_address,
+    async_scanner_count,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Add this:

```py
type HueBLEConfigEntry = ConfigEntry[HueBleLight]

...

async def async_setup_entry(hass: HomeAssistant, entry: HueBLEConfigEntry ) -> bool:

...

async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:

```

Reply:
You marked this as resolved, but you forgot to update the type annotations of `entry` passed to `async_setup_entry` + `async_unload_entry`, please fix that.